packages/cdktf/lib/resource.ts

```diff
@@ -14,6 +14,7 @@ export interface IResourceConstructor<T extends IResource> {
 
 /**
  * A construct which represents a resource.
+ * @deprecated - Please use Construct instead.
```

Comment:
Maybe mention the constructs package as it cannot be imported from cdktf as Resource could?